analytics.html

```diff
@@ -0,0 +1,206 @@
+<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Crimson+Text" />
```

Comment:
So there are a few structural pieces missing in your HTML file. The general structure of a HTML file is:
```
<!DOCTYPE html>
<html>
  <head>
     //meta data about the html file and importing assets generally go here. (ie. loading css, fonts, icon, etc)
  </head>
  <body>
    //all the content of your site goes here.
  </body>
</html>
```